docs/topics/gradle.md

```diff
@@ -316,10 +293,51 @@ kotlin{
 
 </tabs>
 
-> You can use shorthand for a dependency on a Kotlin module, for example, kotlin("test") for "org.jetbrains.kotlin:kotlin-test".
+> You can use shorthand for a dependency on a Kotlin module, for example, kotlin('test') for "org.jetbrains.kotlin:kotlin-test".
```

Comment:
This shorthand has limitations if I remember correctly. If it does, let's write exactly when it's available. Please check

Reply:
This seems to require an effort and out of this feature's scope. Let's do this separately later: https://youtrack.jetbrains.com/issue/KT-45974